### ISSUE TITLE
Missing language flags in Storefront

### ISSUE THREAD

**Issue by chris-dan** [shopware/shopware#295]
**Description**  
Since SW6 EA2 the language selection in the Storefront is missing the flags.
![image](https://user-images.githubusercontent.com/55582279/69124144-69407380-0aa3-11ea-8cb2-bf8fbe65626e.png)

To see the language selection you have to use a Storefront with multiple languages and set multiple domains with different languages.
The current git version still has this bug. In EA1 and EA1.1 it worked fine.

**Possible Solution**  
./.

**Additional context**  
./.

**Comment by tobiasberge** [comment]
Hi Chris,

thanks for reporting this issue. 

This is caused by an association inside `src/Storefront/Pagelet/Header/HeaderPageletLoader.php`.

If you have a look inside the `loadLanguages` method in line 102 you will find this code:
```
$criteria->addAssociation('language.translationCode');
```
This is currently not working and the language code inside the template is `null`. Thats why the flags are not shown because they need a valid language code.

It will be changed to:
```
$criteria->addAssociation('translationCode');
```
**TLDR:**

A fix for this is already in code review and will be in master soon.
It is being fixed along with some other storefront issues. You can follow the ticket progress here: 

https://issues.shopware.com/issues/NEXT-4965

Kind regards
Tobias

**Comment by chris-dan** [comment]
Hi Tobias,

thanks for your answer. I am looking forward to migrate from SW5 to SW6.

**Comment by tobiasberge** [comment]
That's great to hear. 👍 

Ticket NEXT-4965 is resolved and the issue should now be fixed on the latest `master`.

Kind regards
Tobias
